src/lib/Util/adapters/bbc-kaldi/index.js

```diff
@@ -145,35 +34,69 @@ const groupWordsInParagraphs = (words) => {
   return results;
 };
 
+// const groupWordsInParagraphsBySpeakers = (words, speakerSegments) => {
+
+// };
+
+// const getSpeaker = (segments, index) =>{
+
+// }
+
 const bbcKaldiToDraft = (bbcKaldiJson) => {
   const results = [];
   let tmpWords;
+  let speakerSegmentation = null;
+  let wordsByParagraphs =[];
 
   // BBC Octo Labs API Response wraps Kaldi response around retval,
   // while kaldi contains word attribute at root
   if (bbcKaldiJson.retval !== undefined) {
```

Comment:
Why retval? is this just for the specific json fixture or for all inputs? bit of a needless term perhaps?

Reply:
it's from @bevand10 labs API for OCTO/WON it returns the results inside `retval` 🤷‍♂️